remoteworkitem/flatten_test.go

```diff
@@ -0,0 +1,25 @@
+package remoteworkitem
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/almighty/almighty-core/resource"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFlattenMap(t *testing.T) {
+	resource.Require(t, resource.UnitTest)
+	testString := []byte(`{"name":"shoubhik","assignee":{"1":"sbose","2":"pranav","participants":{"4":"sbose56","5":"sbose78"}},"name":"shoubhik"}`)
```

Comment:
Thanks @kwk , I've added that in this static string in the string which gets flattened in the test. In any case arrays were present in the JIRA and Github responses we were already  testing as part of the other unit tests.

However, keys with  arrays are currently not relevant since none of the fields we are considering are arrays.